src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable cow." ) );
```

Comment:
I left it open on purpose to let people mod more animals, like alpacas, goats, and all kinds of mammals.